app/presenters/course_for_menu_presenter.rb

```diff
@@ -31,6 +31,7 @@ def initialize(course, available_section_tabs, user = nil, context = nil)
     @available_section_tabs = (available_section_tabs || []).select do |tab|
       DASHBOARD_CARD_TABS.include?(tab[:id])
     end
+
```

Comment:
don't care that much, but this is a solo change in a file that doesn't change anything :)